EpikCord/client/http.py

```diff
@@ -49,24 +49,69 @@ def __eq__(self, other: Any):
         return False
 
 class Route:
+    """Represents a HTTP route."""
     def __init__(
         self,
         method: str,
         url: str,
         *,
         major_parameters: MajorParameters = MajorParameters()
     ):
+        """
+        Parameters
+        ----------
+        method: str
+            The method of the route.
```

Comment:
Method could be a `Enum` type

Reply:
I do not want to have to do `Route(Methods.GET, URL)` for everything...

Reply:
> I do not want to have to do `Route(Methods.GET, URL)` for everything...

And this is why partials exists